erpnext/hr/utils.py

```diff
@@ -114,16 +114,23 @@ def get_onboarding_details(parent, parenttype):
 		filters={"parent": parent, "parenttype": parenttype},
 		order_by= "idx")
 
-@frappe.whitelist()
-def get_boarding_status(project):
+def update_employee_boarding_status(project):
```

Comment:
Why employee onboarding related functions and common class in hr/utils? Creating a separate file or putting in employee_onboarding.py will make more sense.